vsintegration/tests/unittests/IndentationServiceTests.fs

```diff
@@ -37,36 +39,133 @@ namespace testspace
         static member testmember = 1
 
 "
+
+    static let autoIndentTemplate = "
+let plus x y =
+    x + y
+
+let mutable x = 0
+x <-
+    10 * 2
+
+match Some 10 with
+| None -> 0
+| Some x ->
+    x + 1
+
+try
+    failwith \"fail\"
+with
+    | :? System.Exception -> \"error\"
```

Comment:
Thank you.
As for the line after `with`, it would not be auto indent, because `match x with <enter>` should not.